backend/src/data/migrations/20220807131614_add_groups_to_permissions_table.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+
+enum ColumnName {
+  ID = 'id',
+  CREATED_AT = 'created_at',
+  UPDATED_AT = 'updated_at',
+  IS_ALLOWED = 'is_allowed',
```

Comment:
```suggestion
```
we decided to don't use the flag. just create only relation that we need

don't forget to change DB schema in the root readme